### ISSUE TITLE
Android 9, didn't work

### ISSUE THREAD

**Issue by Dektra00** [nineteendo/pvz2tools#67]
It showed this, I don't know what's the problem, I use the same tools as on Android 8, but when I try it on Android 9, then this happened (I chose template a).
![IMG_20231229_123428](https://github.com/nineteendo/pvz2tools/assets/154240216/c787e066-000d-4476-988d-79c2a5c7d69a)

**Comment by nineteendo** [comment]
Are your sure that you entered a?

**Comment by Dektra00** [comment]
Yes I'm really sure, I tried it almost 10 times and still with the same problem.

**Comment by Dektra00** [comment]
Ohh oke it worked, I use the tools pvz2tools-1.3.0.
But I really wanted it works also on Android 13, for now the tools didn't work for that. I hope u will update it soon.

**Comment by nineteendo** [comment]
I'm afraid I won't be able to update it in the near future:

1. I'm currently studying for the exams.
2. I need to finish a Minecraft video because I haven't uploaded in a long time.
3. I'm making [my own programming language game](http://nineteendo.github.io/tmlx).
4. I want to make a video about Tetris in 1.4 kB on the TI-84.
5. I'm making my own interface library for PyVZ2 with a built-in file manager.
6. I'm refactoring all my code.

**Comment by Dektra00** [comment]
Ahh I can understand, good luck for you.
